### ISSUE TITLE
[A11Y] Split button's split halves are not focusable.

### ISSUE THREAD

**Issue by Lego6245** [microsoft/fluentui#12945]
### Environment Information

- **Package version(s)**: 7.105.X
- **Browser and OS versions**: All

### Describe the issue:

Split buttons do not allow focus to their split halves, requiring Alt + Down to open.

### Please provide a reproduction of the issue in a codepen:

Export to code pen the split button example here: https://developer.microsoft.com/en-us/fluentui#/controls/web/button

Simply tab across the buttons.

#### Actual behavior:

You jump from button to button, never focusing the split half. Alt + Down can open the menu.

#### Expected behavior:

You are able to focus each individual half of the button. A spacebar or enter on either half does the expected behavior.

### Documentation describing expected behavior

https://www.w3.org/TR/wai-aria-1.1/#managingfocus

>  All interactive objects should be focusable. *All parts of composite interactive controls need to be focusable or have a documented alternative method to achieve their function, such as a keyboard shortcut.* Authors are advised to maintain an obvious, discoverable way, either through tabbing or other standard navigation techniques, for keyboard users to move the focus to any interactive element.

Per the aria guidelines, neither condition is met. We have an interactive object that is not focusable, and there is no documented (to the end user) way to interact with it otherwise.

I am aware of previous discussions #6642 and #12394 where the end result was the alt + down usage was required due to focuszone, but if that's the case, this needs to be communicated to the user somehow that this behavior is what's expected. Right now no user with a screen reader would know that alt + down will have that behavior.

**Comment by Lego6245** [comment]
Did some digging in the code, it appears that #6330 added some logic to explicitly prevent this from happening, though I presume that, at the time, this wasn't an intended behavior.

**Comment by joschect** [comment]
@Lego6245 Thanks for doing the work to investigate this! As you found this is intended behavior. @jspurlin or @betrue-final-final might able to comment more but since we provide a hotkey workaround, it's just up to the consumer to pass the appropriate aria-label down detailing how the interaction should work. 

We likely should have more documentation to indicate that this the label is a required property and ensure that users know that they need to pass down this interaction information.

**Comment by Lego6245** [comment]
I'm hesitant to agree that this is the consumer's responsibility to pass this information down, but I suppose from a localization perspective it may not be the role of a UI library to provide it. I'm more curious about the lack of discussion around why both sides of the button are not tab stoppable, something I have not seen a good explanation for other than "that's how it's been".

**Comment by khmakoto** [comment]
Hi @Lego6245, there has been plenty of internal discussion regarding whether a split button should have one or two tab stops, with people taking sides in both camps. To outline the reasonings behind having it being one tab stop is that there is an argument that the split button as a whole is one interactive element that should be focusable in its entirety versus two interactable elements being focused separately as this indicates non-sighted users that there is one control and not two, with the Alt+Down workaround being the default and recommended way to handle menus. Now, I'm on the side of two focus stops being more understandable, but after ample debate we've arrived to the conclusion to have one focus stop.

I hope I've shed some light on our reasons for having split button be as it is and how this was a decision that has been discussed plenty internally. Please let me know if there is anything else I can do to help.

**Comment by Lego6245** [comment]
I understand the reasoning, and while I disagree with it, I'm not sure rocking the boat at this time makes sense. We'll investigate adding a shortcut label to our split buttons.
